docs/articles/documentation/guides/basic-guides/assert.md

```diff
@@ -104,16 +104,11 @@ within a timeout:
 The following web page is an example:
```

Comment:
Please change Line 86 (above) to:

`An additional timeout is usually added when you perform asynchronous functional tests.`

Reply:
Done